src/homework/homework02/iEmployee.java

```diff
@@ -0,0 +1,9 @@
+package homework.homework02;
+
+public interface iEmployee {
```

Comment:
Nazwy interfejsów - z wielkiego I (bo to w sumie nazwa klasy abstrakcyjnej jest)
IEmployee